meson.build

```diff
@@ -40,15 +43,83 @@ postgresql = declare_dependency(
   ],
 )
 
+pgrn_dependencies = [postgresql, groonga, libm]
+
+pgrn_cpp_args = []
+if get_option('msgpack').enabled()
+  msgpack = dependency(get_option('msgpack_package_name'), required: true)
+  pgrn_dependencies += msgpack
+  pgrn_cpp_args += '-DPGRN_HAVE_MSGPACK'
+endif
+
+if get_option('xxhash').enabled()
+  xxhash = declare_dependency(
+    dependencies: [dependency('libxxhash', required: true)],
+    sources: files('vendor/xxHash/xxhash.c'),
+    include_directories: include_directories('vendor/xxHash'),
+  )
+  pgrn_dependencies += xxhash
+endif
+
 pgrn_module_args = {
   'c_args': pgrn_c_args,
-  'dependencies': [postgresql, groonga, libm],
+  'cpp_args': pgrn_cpp_args,
```

Comment:
Is this correct? Is this for C++?

Reply:
fix: [4af310a](https://github.com/pgroonga/pgroonga/pull/750/commits/4af310ada03a8d28ced448a170ba3c0ea85f4f4d) Ahh I totally misunderstood it. Thank you so much! Now, we don't need it because we declare it as a dependecy.